jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamInitRunner.kt

```diff
@@ -51,11 +52,28 @@ class SamInitRunner(
     }
 
     companion object {
+        // TODO: change minimum to 0.7.0 before release
+        private val expectedSamMinVersion = SemVer("0.6.0", 0, 6, 0)
+        private val expectedSamMaxVersion = SemVer("0.8.0", 0, 8, 0)
+
+        fun checkVersion(samVersionLine: String): String? {
```

Comment:
We should apply this validation before any sam invoke.

Reply:
done